app/src/pages/nanomine/contactus/contactus.js

```diff
@@ -0,0 +1,83 @@
+export default {
+  name: 'Contact',
+  data () {
+    return {
+      name: null,
+      email: null,
+      contactType: null,
+      platform: null,
+      message: null,
+      errors: []
+    }
+  },
+  methods: {
+    validateForm: function (e) {
+      this.errors = []
+      if (!this.name) {
+        this.errors.push('Name required')
+      }
+      if (!this.email) {
+        this.errors.push('Email required')
+      } else if (!this.validEmail(this.email)) {
+        this.errors.push('Valid email required.')
+      }
+
+      if (!this.contactType) {
+        this.errors.push('Contact type required')
+      }
+
+      if (!this.message || this.message.trim() === '') {
+        this.errors.push('Message required')
+      }
+
+      if (!this.errors.length) {
+        return true
+      }
+      e.preventDefault()
+    },
+    validEmail: function (email) {
+      const re =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      return re.test(email)
+    },
+
+    resetForm: function () {
+      this.name = null
+      this.email = null
+      this.message = null
+      this.contactType = null
+      this.message = null
+      this.errors = []
+    },
+
+    onSubmit: async function (e) {
+      e.preventDefault()
```

Comment:
I'll suggest since we don't want to conditionally prevent default, we can add the `.prevent` modifier to the form and just remove the `e` and `e.preventDefault` entirely.

Reply:
done ✅